_translator-files/po/de/Getting-Started.po

```diff
@@ -290,7 +258,5 @@ msgstr "## Fehlerbehebung"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:90
-#, fuzzy
-#| msgid "Having audio trouble? Can't see servers, or some other issue? Have a look at the [Troubleshooting page](/wiki/Client-Troubleshooting), or feel free to ask in the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)."
 msgid "Having audio trouble? Can't see Servers, or some other issue? Have a look at the [Troubleshooting page](/wiki/Client-Troubleshooting), or feel free to ask in the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)."
-msgstr "Hast du Probleme mit dem Audio? Kannst du die Serverliste nicht sehen, oder irgendein anderes Problem? Schau dir die [Fehlerbehebungsseite](/wiki/Client-Troubleshooting) an, oder frage gerne im [Diskussionsforum](https://github.com/jamulussoftware/jamulus/discussions)."
+msgstr "Hast du Probleme mit dem Audio? Kannst du die Serverliste nicht sehen, oder irgendein anderes Problem? Schau dir die [Fehlerbehebungsseite](Client-Troubleshooting) an, oder frage gerne im [Diskussionsforum](https://github.com/jamulussoftware/jamulus/discussions)."
```

Comment:
Is it really Client-Troubleshooting not /wiki/Client-Troubleshooting here?

Reply:
Corrected